app/javascript/find/controllers/locations_autocomplete_controller.js

```diff
@@ -18,6 +18,7 @@ export default class extends Controller {
       showNoOptionsFound: false,
       name: inputElement.name,
       defaultValue: inputElement.value,
+      displayMenu: 'overlay',
```

Comment:
This was invading space on the form everytime the suggestions appear so focring the overlay